dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMain.java

```diff
@@ -100,7 +101,7 @@ public static void main(String[] arguments)
         }
 
         if (classPathResult != null) {
```

Comment:
if `classPathResult` is not null, then `classPathBuilder` is not null.

<img width="796" alt="Screen Shot 2020-09-10 at 13 37 36" src="https://user-images.githubusercontent.com/28604/92773727-d8f15900-f36a-11ea-9664-7b9fa25d741c.png">